.travis.yml

```diff
@@ -1,13 +1,38 @@
 sudo: false
+
 language: python
+
 cache: pip 
+
 python:
   - 3.5
+
+env:
+  - PYDATA=true
+  - PYDATA=false CHECK_MIGRATION=true
+
+before_install:
+  - git config --global user.email "you@example.com"
+  - git config --global user.name "Your Name"
```

Comment:
git refuses to pull a branch without these configs. It is probably because git needs to add a merge commit and it requires credentials.